src/Registry/TypeRegistry.php

```diff
@@ -1271,7 +1272,8 @@ public function register_mutation( $mutation_name, $config ) {
 
 		$input_fields = [
 			'clientMutationId' => [
-				'type' => 'String',
+				'type'        => 'String',
+				'description' => 'The ID for the mutation input field',
```

Comment:
translation

Reply:
Let's also update this one to be: 

`This is an ID that can be passed to a mutation by the client to track the progress of mutations and catch possible duplicate mutation submissions.`